src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Common/Icons - ContactJournal.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
+  "images" : [
+    {
+      "filename" : "cross_light.pdf",
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "filename" : "cross_dark.pdf",
+      "idiom" : "universal"
```

Comment:
Can we keep filename matching asset name?

Reply:
Done